packages/components/src/components/feedback/Skeleton/Skeleton.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { Skeleton, SkeletonCircle, SkeletonProps, SkeletonText } from './index';
+import { Flex } from '@components/layouts';
+
+export default {
+  title: 'Components/Feedback/Skeleton',
+} as Meta;
+
+const commonArgs = {
+  children: 'content',
+  isLoaded: false,
+};
+
+const SkeletonTemplate = (args: SkeletonProps) => <Skeleton {...args} />;
```

Comment:
we should remove the `Skeleton` _prefix_ from all stories for consistency with other parts of the code